examples/pyqt5_opencv/app/camera.py

```diff
@@ -0,0 +1,54 @@
+from PyQt5.QtCore import pyqtSignal
+from PyQt5.QtCore import QObject
+from PyQt5.QtCore import QTimer
+from PyQt5.QtGui import QImage
+import cv2
+
+
+class CameraDevice(QObject):
+
+    frame_ready = pyqtSignal(QImage)
+
+    def __init__(self, device_id=-1):
+        super().__init__()
+        self.capture = cv2.VideoCapture(device_id)
+        self.timer = QTimer()
+
+        if not self.capture.isOpened():
+            raise ValueError("Device not found")
+
+        self.timer.timeout.connect(self.read_frame)
+        self.timer.setInterval(1000/self.fps)
```

Comment:
... this value to `(1000/(self.fps or 30))`. Then I got images from my webcam.

Does that work for you?